bears/general/CPDBear.py

```diff
@@ -0,0 +1,41 @@
+import re
+import os
+
+from coalib.bearlib.abstractions.Lint import Lint
+from coalib.bears.LocalBear import LocalBear
+
+
+class CPDBear(LocalBear, Lint):
```

Comment:
Hope so, I haven't used the GlobalBear until now, but I will do my best to understand it